ironfish-cli/src/commands/wallet/transaction/index.ts

```diff
@@ -80,10 +80,6 @@ export class TransactionCommand extends IronfishCommand {
         memo: {
           header: 'Memo',
         },
```

Comment:
I am guessing this is related to the deprecation? Just verifying.

Reply:
Yeah, it's not totally necessary to remove this here, but I thought we should remove our own usage of the field if we're deprecating it 😁